package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "build": "node ./build.js ./build",
     "serve": "node ./server.js ./build 3000",
-    "test": "npm run build && npm run serve"
+    "test": "npm run build && npm run serve",
+    "hot": "./node_modules/.bin/nodemon --exec npm test"
```

Comment:
just FYI: if package is available in `./node_modules/.bin` you can write tool name to achieve exactly same effect. In other words: `"hot": "nodemon --exec npm test"` does exactly same

Reply:
Resolved with commit 8fbe48e